index.fr.md

```diff
@@ -0,0 +1,147 @@
+-   les utilisateurs peuvent avoir plus de travail pour compenser des lacunes dans les fonctionnalités d’accessibilité dans les navigateurs, les lecteurs de média, et la technologie d’assistance et le manque d’accessibilité du contenu&nbsp;; par exemple en utilisant différents navigateurs ou technologies d’assistance pour surmonter différents problèmes.
+ 
+Cependant dans la plupart des cas les contournements ne sont pas mis en œuvre et le résultat reste décevant en termes d’accessibilité. De plus un support défaillant de l’accessibilité dans une composante ne peut parfois pas être raisonnablement compensé par les autres composantes et le résultat en est l’inaccessibilité, rendant impossible pour certaines personnes handicapées l’utilisation d’un site, d’une page ou d’une fonctionnalité précis.
+ 
+## Règles et autres standards {#guidelines}
+ 
+L’Initiative pour l’Accessibilité du Web (<span lang="en">Web Accessibility Initiative</span> ([WAI](https://www.w3.org/WAI/)) du <span lang="en">World Wide Web Consortium</span> ([W3C {% include_cached different.html %}](https://www.w3.org/)) développe des **standards d’accessibilité du web** pour différentes composantes&nbsp;:
+ 
+-   [les règles pour l'accessibilité des outils d'édition / <span lang="en">Authoring Tool Accessibility Guidelines</span> (ATAG)](/standards-guidelines/atag/) traite des outils de création&nbsp;;
+-   [les règles pour l'accessibilité des contenus Web / <span lang="en">Web Content Accessibility Guidelines</span> (WCAG)](/standards-guidelines/wcag/) traite du contenu web, et est utilisé par les créateurs, outils de création et outils d’évaluation de l’accessibilité&nbsp;;
+-   [les règles pour l'accessibilité des agents utilisateurs / <span lang="en">User Agent Accessibility Guidelines</span> (UAAG)](/standards-guidelines/uaag/) traite des navigateurs et lecteurs de média, y compris certains aspects des technologies d’assistance.
+ 
+Ces règles d’accessibilité reposent sur le socle des spécifications techniques fondamentales du Web, et sont développées en coordination avec toutes les <a href="https://www.w3.org/TR/">spécifications techniques du W3C {% include_cached different.html %}</a> (HTML, CSS, SVG, SMIL, etc.). Le W3C développe aussi des spécifications techniques qui traitent directement d’accessibilité, dont&nbsp;:
+ 
+* la suite des spécifications [ARIA, Les applications Internet riches accessibles / <span lang="en">The Accessible Rich Internet Applications</span>](/standards-guidelines/aria/), qui définit une façon de créer des applications web plus accessibles aux personnes handicapées. Elle facilite notamment le développement de contenu dynamique et de contrôles d’interface utilisateur avancés avec Ajax, HTML, JavaScript et les technologies associées.
```

Comment:
```suggestion
* la suite des spécifications [<abbr lang="en" title="Accessible Rich Internet Applications">ARIA</abbr>, les applications Internet riches accessibles</span>](/standards-guidelines/aria/), qui définit une façon de créer des applications web plus accessibles aux personnes en situation de handicap. Elle facilite notamment le développement de contenu dynamique et de contrôles d’interface utilisateur avancés avec Ajax, HTML, JavaScript et les technologies associées.
```

Reply:
same comment cf https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr

Reply:
I disagree. The French translation of the UN Convention is « Convention relative aux droits des personnes **handicapées**  » cf
https://www.ohchr.org/fr/instruments-mechanisms/instruments/convention-rights-persons-disabilities
You can be in a « situation de handicap » without any disability. It’s a expression from the the domain of ergonomy.

Reply:
See https://github.com/w3c/wai-components/pull/16#discussion_r1377976104